inference-engine/tests/functional/plugin/gpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -49,5 +49,8 @@ std::vector<std::string> disabledTestPatterns() {
             R"(.*(smoke_DetectionOutput3In).*)",
             R"(.*(smoke_DetectionOutput5In).*)",
             R"(.*(ScatterUpdateLayerTest).*)",
+
+            // INT8 StridedSlice not supported
+            R"(.*(StridedSliceTransformation).*)",
```

Comment:
is there any LPT Prefixes? If yes, please add

Reply:
Thanks, done